server/seedHelpers.js

```diff
@@ -0,0 +1,79 @@
+const db = require('./db.js');
+const faker = require('faker');
+
+// HELPER FUNCTIONS
+const randomCondition = () => {
+  const conditions = ['New', 'Mint', 'Used', 'Fair'];
+  return conditions[faker.random.number(3)];
+};
+
+// Roughly half the products should have notes from the seller
+const sellerNote = () => {
+  return Math.random() > 0.5 ? faker.lorem.sentence() : '';
+};
+
+// Returns a URL-friendly version of the given string. For example,
+// 'John O'Riley' => 'john-oriley'.
+// This function helps produce names that can be used in API requests.
+const urlify = (string) => {
+  return string.toLowerCase()
+    .replace(/\s/g, '-')
+    .replace(/[^a-zA-Z\-\d]/g, ''); // strip out any character that is not a letter, digit, or hyphen
+};
+
+// Roughly 80% of the products should be returnable
+const returnsAllowed = () => {
+  return Math.random() < 0.8;
+};
+
+// Returns an object representing a fake product
+const generateProduct = (id) => {
+  const name = faker.commerce.productName();
+
+  return {
+    id,
+    name,
+    'url-name': urlify(name),
+    condition: randomCondition(),
+    price: parseFloat(faker.commerce.price()),
+    sellerNote: sellerNote(),
+    expiresAt: faker.date.recent(-30), // a date up to 30 days in the future
+    watchers: faker.random.number(75),
+    bids: faker.random.number(50),
+    shippingCountry: faker.address.country(),
+    returnsAllowed: returnsAllowed()
+  };
+};
+
+// MAIN FUNCTIONS
+// This function generates fake products and adds them to the database. It
+// returns a promise that resolves once all fake products have been added to the
+// database.
+const seed = (startId, endId) => {
+  console.log('Seeding database...');
```

Comment:
I'd probably put these `console.log`s btwn lines 69 and 70 so that they're all visually in the same spot.